plugins/alignments/src/LinearPileupDisplay/components/ColorByTag.tsx

```diff
@@ -47,33 +48,41 @@ export default function ColorByTagDlg(props: {
           <CloseIcon />
         </IconButton>
       </DialogTitle>
-      <DialogContent>
-        <Typography>Set the tag to color by</Typography>
+      <DialogContent style={{ overflowX: 'hidden' }}>
         <div className={classes.root}>
+          <Typography>Enter tag to color by</Typography>
           <form>
             <TextField
-              id="standard-select-currency"
-              select
               value={tag}
               onChange={event => {
                 setTag(event.target.value)
               }}
-            >
-              <MenuItem value="" />
-              <MenuItem value="HP">HP (haplotype)</MenuItem>
-              <MenuItem value="XS">XS (RNA-seq strandedness)</MenuItem>
-              <MenuItem value="TS">TS (RNA-seq strandedness)</MenuItem>
-              <MenuItem value="YC">YC (color encoded)</MenuItem>
-            </TextField>
+              placeholder="Enter tag name"
+              inputProps={{
+                maxLength: 2,
+                'data-testid': 'color-tag-name-input',
+              }}
+              error={tag.length === 2 && !validTag}
+              helperText={
+                tag.length === 2 && !validTag ? 'Not a valid tag' : ''
+              }
+              autoComplete="off"
+              data-testid="color-tag-name"
+            />
             <Button
-              onClick={() => {
-                const display = model.displays[0]
-                ;(display.PileupDisplay || display).setColorScheme({
+              variant="contained"
+              color="primary"
+              type="submit"
+              style={{ marginLeft: 20 }}
+              onClick={async () => {
+                const display = await model.displays[0]
```

Comment:
any reason why this needs `await model.displays[0]`?

Reply:
I dont think so, I think i left it in when the function wasn't working and this was one of the things I tested. removing it now